src/main/java/twilightforest/block/BlockTFPortal.java

```diff
@@ -89,10 +90,17 @@ public void addCollisionBoxToList(IBlockState state, World world, BlockPos pos,
 	}
 
 	@Override
-	public boolean isOpaqueCube(IBlockState state) {
```

Comment:
do we need to override both full/opaque?

Reply:
`isOpaqueCube` is handled by the `BlockBreakable` superclass here.